knimin/lib/tests/test_shotgun.py

```diff
@@ -24,7 +24,21 @@ def setUp(self):
                                  [7.86, 8.07, 8.16, 9.64],
                                  [12.29, 7.64, 7.32, 13.74]])
 
+        self.cp_vals_nan = np.array([[10.14, 7.89, 7.9, 15.48],
+                                     [7.86, 8.07, np.nan, 9.64],
+                                     [12.29, 7.64, np.nan, 13.74]])
+
         self.qpcr_conc = \
+            np.array([[39.25850585, 195.12485652, 193.73923464, 0.87336277],
+                      [199.34146596, 171.63359148, 160.97081284, 56.0640694],
+                      [8.48213356, 233.17824124, 292.90620164, 3.0180584]])
+
+        self.qpcr_conc_nan = \
+            np.array([[39.25850585, 195.12485652, 193.73923464, 0.87336277],
+                      [199.34146596, 171.63359148, 0, 56.0640694],
+                      [8.48213356, 233.17824124, 0, 3.0180584]])
+
+        self.qpcr_conc_25000 = \
```

Comment:
given the default change with `compute_qpcr_concentration`, should this variable name be 10000?